status-im/status-react
# [#4320] Do not inject jQuery in DApps browser

fixes #4320

### Summary:

Do not always inject jQuery in DApps browser.

### Steps to test:
- Open Status
- Make sure all featured DApps work as expected :sweat_smile: 

status: ready
